components/RoastDetailCard/index.js

```diff
@@ -38,51 +34,35 @@ export default function RoastDetailCard({
         height={""}
         alt="Coffee-Package"
       />
-      {!edit && (
-        <StyledSectionVariant>
-          <StyledHeading>{name}</StyledHeading>
-          <StyledParagraph>{roaster}</StyledParagraph>
-          <StyledHeading>Arabica / Robusta</StyledHeading>
-          <StyledParagraph>
-            {arabica} / {robusta}
-          </StyledParagraph>
-          <StyledHeading>Röstgrad</StyledHeading>
-          <StyledParagraph>{level}</StyledParagraph>
-          <StyledHeading>Herkunft</StyledHeading>
-          <StyledParagraph>{provenance}</StyledParagraph>
-          <StyledRating>
-            Bewertung:{" "}
-            {score.length > 0
-              ? Math.floor(
-                  score.reduce((acc, curr) => acc + curr, 0) / score.length
-                )
-              : 0}
-            /100
-          </StyledRating>
-          <StyledNumberRating>
-            {score.length} {score.length === 1 ? "Bewertung" : "Bewertungen"}
-          </StyledNumberRating>
-        </StyledSectionVariant>
-      )}
-      {edit && (
-        <EditForm
-          onSubmit={onSubmit}
-          name={name}
-          roaster={roaster}
-          provenance={provenance}
-          arabica={arabica}
-          robusta={robusta}
-          level={level}
-        />
-      )}
+      <StyledSectionVariant>
+        <StyledHeading>{name}</StyledHeading>
+        <StyledParagraph>{roaster}</StyledParagraph>
+        <StyledHeading>Arabica / Robusta</StyledHeading>
+        <StyledParagraph>
+          {arabica} / {robusta}
+        </StyledParagraph>
+        <StyledHeading>Röstgrad</StyledHeading>
+        <StyledParagraph>{level}</StyledParagraph>
+        <StyledHeading>Herkunft</StyledHeading>
+        <StyledParagraph>{provenance}</StyledParagraph>
+        <StyledRating>
+          Bewertung:{" "}
+          {score.length > 0
+            ? Math.floor(
+                score.reduce((acc, curr) => acc + curr, 0) / score.length
+              )
+            : 0}
+          /100
```

Comment:
if you want to be perfect with the readability, you could refactor this into a ulitily function called "calculateScore"